gradle/libs.versions.toml

```diff
@@ -0,0 +1,20 @@
+[versions]
+kotlin-sdk = "1.9.22"
+kotlinx-benchmark = "0.4.10"
+kotlinx-validator = "0.13.2"
+guava-testlib = "18.0"
+jmh = "1.21"
```

Comment:
New version catalog; other than updating `kotlin-sdk` to `1.9.22`, no version changes were applied.